cli/alsatian-cli.ts

```diff
@@ -1,18 +1,37 @@
 #! /usr/bin/env node
 
-import { createTestSet, TestRunner } from "../core/alsatian-core";
+import { createTestSet, TestRunner, TestOutput } from "../core/alsatian-core";
 import { CliTestRunner } from "./cli-test-runner";
 import { AlsatianCliOptions } from "./alsatian-cli-options";
 
+const {
+  TapBark
+} = require("tap-bark");
+
 // get all arguments from the user
 let userArguments = new AlsatianCliOptions(process.argv.slice(2));
 
 // create test set from given file globs
 let testSet = createTestSet();
 testSet.addTestsFromFiles(userArguments.fileGlobs);
 
+let reporter: any;
+
+if (userArguments.tap) {
```

Comment:
test coverage has gone down :( looks like here is the issue - we have no tests here yet so may need a bit of work to write some tests around here. Will need them pre beta though so ideal to do them if possible. If too difficult no worries I'll do them later